### ISSUE TITLE
Add an error message for missing form fields 

### ISSUE THREAD

**Issue by s-gbz** [codeklasse/codeklasse.de#39]
As of right now, we have a browser provided hint mechanism if some form field is missing.

![Screenshot from 2022-01-19 20-37-56](https://user-images.githubusercontent.com/23424538/150202681-25f9673c-18cf-491f-ab43-ecce43dcc09d.png)

It would be nice to print a message below the send button in that case.
The message would say "Bitte füllen Sie alle Felder aus" ("Please fill in all fields" in German).
Also, the fields would be highlighted red via 

```html
<input class="input is-danger" type="text" placeholder="Danger input">
```

Take a look at the Bulma [docs](https://bulma.io/documentation/form/input/) for reference.
The message would disappear if all fields were filled out afterwards.
The JS code should be free of libraries and be written inline at the bottom of the contact form [file](https://github.com/codeklasse/codeklasse.de/blob/master/_includes/contact-form.html)

**Comment by s-gbz** [comment]
Sure thing @pramod74!
Here are some references  (- feel free to ignore them just as well)

- https://github.com/prichmp/bulma-form-validation
- https://www.ma-no.org/en/programming/javascript/validating-html-forms-using-bulma-and-vanilla-javascript

**Comment by seann1** [comment]
When I clone this project and run "bundle install" I get this error: An error occurred while installing http_parser.rb (0.8.0), and Bundler
cannot continue.

**Comment by pramod74** [comment]
> Sure thing @pramod74! Here are some references (- feel free to ignore them just as well)
> 
> * https://github.com/prichmp/bulma-form-validation
> * https://www.ma-no.org/en/programming/javascript/validating-html-forms-using-bulma-and-vanilla-javascript

Thanks, @s-gbz. I am reviewing the docs, and shall revert soon.

**Comment by s-gbz** [comment]
Closed by #44
